src/views/homepage/components/pre-footer/pre-footer.module.css

```diff
@@ -0,0 +1,67 @@
+.preFooter {
+  --container-max-width: 1140px;
+  --grid-template-columns: 4;
+  --grid-column-gap: 24px;
+
+  position: relative;
+  padding: 64px 24px;
+  background-image: radial-gradient(
+    at bottom,
+    var(--token-color-palette-neutral-100),
+    --token-color-palette-neutral-0
+  );
+
+  @media (--medium-up) {
+    --grid-template-columns: 12;
+    --grid-column-gap: 32px;
+  }
+}
+
+.container {
+  max-width: var(--container-max-width);
+  margin-right: auto;
+  margin-left: auto;
+  display: grid;
+  grid-template-columns: repeat(var(--grid-template-columns), 1fr);
+  grid-column-gap: var(--grid-column-gap);
+  grid-row-gap: 32px;
+  align-items: center;
+}
+
+.content {
+  grid-column: 1 / -1;
+
+  @media (--medium-up) {
+    grid-column: 2 / 6;
```

Comment:
Nit: on large viewports, the `.content` looks a little more width-constrained than in the design spec:

<img width="1440" alt="CleanShot 2022-04-12 at 11 47 21@2x" src="https://user-images.githubusercontent.com/4624598/163002348-06fbd862-544a-4331-80e2-7c0e9d8ce19e.png">



vs design spec (at least, I _think_ this is the spec, wasn't sure as there were many similar artboards):

<img width="1473" alt="CleanShot 2022-04-12 at 11 38 10@2x" src="https://user-images.githubusercontent.com/4624598/163000669-a5f4b98b-52e6-4b2c-a587-3fbb1c32f15a.png">

Not exactly sure how to adjust (if we want to adjust) - maybe:

```suggestion
    grid-column: 2 / 7;
```

which yields:

<img width="1439" alt="CleanShot 2022-04-12 at 11 47 42@2x" src="https://user-images.githubusercontent.com/4624598/163002327-13ffb01c-f9e0-4b95-8925-7df2ce83285b.png">


TBH I'm not familiar with the grid being used for dev-dot designs (😬) so I might be completely off here 👍